relay-event-normalization/src/transactions/processor.rs

```diff
@@ -1362,6 +1326,11 @@ mod tests {
     },
 ]"#);
 
+        assert_eq!(
```

Comment:
These assertions on the transaction source were lost in a previous PR, adding them back in this one.